cli/generator-op-microservice/generators/microservice/templates/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "<%= serviceName %>",
+  "description": "<%= serviceClassName %> service for One Platform",
+  "opservice": {
+    "className": "<%= serviceClassName %>",
+    "serviceName": "<%= serviceName %>"
+  },
+  "version": "<%= serviceVersion %>",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "github:1-Platform/one-platform",
+    "directory": "packages/<%= packageName %>"
+  },
+  "publishConfig": {
+    "access": "private"
+  },
+  "private": true,
+  "files": [ "dist" ],
+  "scripts": {
+    "test": "jest --config=jest.config.json --detectOpenHandles --forceExit",
```

Comment:
If you change the jest.config file format as a js file, you don't need to add a config flag. It takes the `jest.config.js` file from the nearest parent directory as its config.

Reply:
Also, can you find out why the tests don't exit normally instead of using --forceExit?

Excerpt from [jest docs](https://jestjs.io/docs/en/cli.html#--forceexit):
> Note: This feature is an escape-hatch. If Jest doesn't exit at the end of a test run, it means external resources are still being held on to or timers are still pending in your code. It is advised to tear down external resources after each test to make sure Jest can shut down cleanly. You can use --detectOpenHandles to help track it down.